src/sass/Fabric.scss

```diff
@@ -9,6 +9,7 @@
 @import './Fabric.Utilities.Output';
 @import './Fabric.ZIndex.Variables';
 @import './Fabric.Mixins';
+@import './Fabric.i18n';
```

Comment:
We have other files with internationalization/localization mixins. Maybe we should namespace these (e.g. `Fabric.Mixins.Localization.RTL.scss`) or place them in subfolders.